recipes-support/host-insight-helper/host-insight-helper/host-insight-helper.sh

```diff
@@ -1,17 +1,40 @@
 #!/bin/sh
 
 ID_PATH="/etc/opt/host-insight-client/identity-fallback.toml"
+CLIENT_UPGRADE="/tmp/host-insight/client_upgrade"
```

Comment:
who is creating this file? and chould not assign_uid_and_fallback_domain be runninng after upgrade_client?

Reply:
It gets created by the client's exit handler as [suggested here](https://github.com/hostmobility/host-insight-client/pull/4/files#diff-3e630e504435357be943f42ac4dcb0a964e31fccda6848395f01c6404e96bd90).

As for your second question, I think not because the helper service is executed at first boot  and then disables itself.

Reply:
perfect then this is no issue.